kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -37,36 +37,42 @@
           />
         </KGridItem>
         <KGridItem sizes="100, 100, 50" percentage>
-          <KTextbox
-            ref="questionsInput"
-            v-model.trim.number="numQuestions"
-            type="number"
-            :min="1"
-            :max="maxQs"
-            :invalidText="$tr('numQuestionsBetween')"
-            :invalid="numQuestionsInvalid"
-            :label="$tr('numQuestions')"
-            class="number-field"
-            @blur="numQuestionsBlurred = true"
-          />
-          <UiIconButton
-            type="flat"
-            aria-hidden="true"
-            class="number-btn"
-            :disabled="numQuestions === 1"
-            @click="numQuestions -= 1"
-          >
-            <mat-svg name="remove" category="content" />
-          </UiIconButton>
-          <UiIconButton
-            type="flat"
-            aria-hidden="true"
-            class="number-btn"
-            :disabled="numQuestions === maxQs"
-            @click="numQuestions += 1"
-          >
-            <mat-svg name="add" category="content" />
-          </UiIconButton>
+          <KGrid>
+            <KGridItem sizes="100, 100, 75" percentage>
+              <KTextbox
+                ref="questionsInput"
+                v-model.trim.number="numQuestions"
+                type="number"
+                :min="1"
+                :max="maxQs"
+                :invalid="Boolean(showError && numQuestIsInvalidText)"
+                :invalidText="numQuestIsInvalidText"
+                :label="$tr('numQuestions')"
+                class="number-field"
+                @blur="numQuestionsBlurred = true"
+              />
+            </KGridItem>
+            <KGridItem sizes="100, 100, 25" percentage>
+              <UiIconButton
+                type="flat"
+                aria-hidden="true"
+                class="number-btn"
+                :disabled="numQuestions === 1"
+                @click="numQuestions -= 1"
+              >
+                <mat-svg name="remove" category="content" />
+              </UiIconButton>
+              <UiIconButton
+                type="flat"
+                aria-hidden="true"
+                class="number-btn"
+                :disabled="numQuestions === maxQs"
+                @click="numQuestions += 1"
+              >
+                <mat-svg name="add" category="content" />
+              </UiIconButton>
+            </KGridItem>
+          </KGrid>
```

Comment:
what was the intent of this change? This is what I'm seeing:

| before | after |
|--|--|
| ![image](https://user-images.githubusercontent.com/2367265/54853360-eb765f00-4cac-11e9-988c-91a442dd573d.png) |  ![image](https://user-images.githubusercontent.com/2367265/54853322-cd106380-4cac-11e9-9f8e-99ccf3c5b075.png) |

Reply:
The aim was to give the text field a consistent width, as now that the validation messages are showing properly, the `after` view was being shown consistently without this change, but only when the validation text appeared.

Reply:
I also only tested this on a wider form factor, so happy for feedback on how to handle this.